pkg/postgres/schema/compliance_operator_scan_v2.go

```diff
@@ -54,13 +54,13 @@ const (
 
 // ComplianceOperatorScanV2 holds the Gorm model for Postgres table `compliance_operator_scan_v2`.
 type ComplianceOperatorScanV2 struct {
-	ID               string     `gorm:"column:id;type:varchar;primaryKey"`
-	ScanConfigName   string     `gorm:"column:scanconfigname;type:varchar"`
-	ClusterID        string     `gorm:"column:clusterid;type:uuid;index:complianceoperatorscanv2_sac_filter,type:hash"`
-	ProfileProfileID string     `gorm:"column:profile_profileid;type:uuid"`
-	StatusResult     string     `gorm:"column:status_result;type:varchar"`
-	LastExecutedTime *time.Time `gorm:"column:lastexecutedtime;type:timestamp"`
-	ScanName         string     `gorm:"column:scanname;type:varchar"`
-	ScanRefID        string     `gorm:"column:scanrefid;type:uuid"`
-	Serialized       []byte     `gorm:"column:serialized;type:bytea"`
+	ID                  string     `gorm:"column:id;type:varchar;primaryKey"`
+	ScanConfigName      string     `gorm:"column:scanconfigname;type:varchar"`
+	ClusterID           string     `gorm:"column:clusterid;type:uuid;index:complianceoperatorscanv2_sac_filter,type:hash"`
+	ProfileProfileRefID string     `gorm:"column:profile_profilerefid;type:uuid"`
```

Comment:
Since this replaces a column, does that cause any issues with a rollback to 4.4? Or did `ProfileProfileID` column not exist in 4.4 and was added early on in 4.5 development?

Reply:
I see. Rollback shouldn't be a problem then as the `profile_profileid` column would not be removed on upgrade.